core/server/master/src/main/java/alluxio/master/meta/AlluxioMasterRestServiceHandler.java

```diff
@@ -848,6 +855,36 @@ public Response getWebUIWorkers() {
     }, Configuration.global());
   }
 
+  /**
+   * Gets Web UI Master page data.
+   *
+   * @return the response object
+   */
+  @GET
+  @Path(WEBUI_MASTERS)
+  public Response getWebUIMasters() {
+    return RestUtils.call(() -> {
+      MasterWebUIMasters response = new MasterWebUIMasters();
+
+      response.setDebug(Configuration.getBoolean(PropertyKey.DEBUG));
+
+      List<MasterInfo> lostMasterInfos = mMetaMaster.getLostMasterInfoList();
+      MasterInfo[] failedMasterInfos = WebUtils.generateMasterInfos(lostMasterInfos);
+      response.setFailedMasterInfos(failedMasterInfos);
```

Comment:
can the response take List instead of Array? If not, `mMetaMaster.getLostMasterInfo()` can just return an array?